src/libs/ReceiptUtils.ts

```diff
@@ -66,7 +66,8 @@ function getThumbnailAndImageURIs(transaction: Transaction, receiptPath: string
         image = ReceiptSVG;
     }
 
-    return {thumbnail: image, image: path, isLocalFile: true};
+    const isLocalFile = path.startsWith('blob:') || path.startsWith('file:');
```

Comment:
This caused regression. iOS app crashes on distance request.

Here's fix:
```suggestion
    const isLocalFile = typeof path === 'number' || path.startsWith('blob:') || path.startsWith('file:');
```

![Simulator Screenshot - iPhone 15 - 2024-01-04](https://github.com/Expensify/App/assets/108292595/5b1a8cc6-9105-4e89-a7fd-05fa512aba1c)

Reply:
Just curious, In what case it is number?

Reply:
In native, `source` is number for the local images

https://reactnative.dev/docs/image#imagesource

<img width="659" alt="Screenshot 2024-01-04 at 11 39 14 AM" src="https://github.com/Expensify/App/assets/108292595/55c9af43-eaf7-4405-87b4-6daaab0465e6">

Reply:
Thanks for clarifying!